Lib/test/test_httpservers.py

```diff
@@ -1535,6 +1538,158 @@ def test_server_test_ipv4(self, _):
             server.test(ServerClass=mock_server, bind=bind)
             self.assertEqual(mock_server.address_family, socket.AF_INET)
 
+class CommandLineTestCase(unittest.TestCase):
+    def setUp(self):
+        self.default_port = 8000
+        self.default_bind = None
+        self.default_protocol = 'HTTP/1.0'
+        self.default_handler = SimpleHTTPRequestHandler
+        self.default_server = http.server.CommandLineServerClass
+        self.tls_cert = '-----BEGIN CERTIFICATE-----\n' + '-----END CERTIFICATE-----\n'
+        self.tls_key = '-----BEGIN RSA PRIVATE KEY-----\n' + '-----END RSA PRIVATE KEY-----\n'
+
+        self.tls_password = '<PASSWORD>'
```

Comment:
if you need "tls" files you can find them in `./Lib/test/certdata` dir
you can also look at `BaseHTTPSServerTestCase` as an example of testing https protocol:
https://github.com/python/cpython/blob/d19af00b90d94cd987293c479b8c3ef698bf7d3e/Lib/test/test_httpservers.py#L366-L378